upf/upf_pfcp.c

```diff
@@ -1437,7 +1437,7 @@ static void
 compile_sdf (int is_ip4, const upf_pdr_t * pdr,
 	     const acl_rule_t * rule, upf_acl_t * acl)
 {
-  if (!(pdr->pdi.fields & F_PDI_SDF_FILTER))
+  if (!(pdr->pdi.fields & (F_PDI_SDF_FILTER | F_PDI_APPLICATION_ID)))
```

Comment:
I think this test can go or make it an `ASSERT`

Reply:
done